client/src/App.jsx

```diff
@@ -15,14 +15,16 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.getFeaturedMovie('feature=true');
+    let qs = new URLSearchParams(window.location.search);
+    this.getFeaturedMovie(qs.get('id'));
```

Comment:
might want to set a default option in case the user doesn't add a query string

Reply:
Will, i'm not sure why I'm getting this error running my proxy.  FetchError: invalid json response body at http://localhost:3002/api/featured?id=1 reason: Unexpected end of JSON input

Reply:
@glkapitan You may want to add some `console.log`s to see what data you are actually receiving - you are likely receiving an empty request or data that is not JSON.